topo_processor/cli/main_test.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+import shutil
+from tempfile import gettempdir
+
+import pytest
+
+from topo_processor.metadata import DataType, create_collection
+
+from .main import upload_to_local_disk
+
+
+@pytest.fixture(autouse=True)
+def teardown():
+    yield
```

Comment:
why the `yeild` if you are going to do something werid, its generally a good idea to comment it as why